indexer/os/Linux.cc

```diff
@@ -107,6 +110,20 @@ bool setCurrentThreadName(std::string_view name) {
   return retCode == 0;
 }
 
+std::variant<uint64_t, std::error_code> availableSpaceForIpc() {
+  struct statvfs shm_info;
+  // Q: Is boost/interprocess guaranteed to use /dev/shm on Linux?
+  // It seems to be the case, but I haven't yet been able to figure
+  // out where the path is decided.
```

Comment:
boost/interprocess used SYSV shared memory. /dev/shm seems to be guaranteed for Linux distros using glibc 2.2+ for POSIX shared memory. https://sourcegraph.com/github.com/torvalds/linux/-/blob/Documentation/filesystems/tmpfs.rst?L41-50

Can't tell if it's possible that the two don't line up in some situations...